qiskit/providers/experiment/constants.py

```diff
@@ -0,0 +1,23 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Experiment constants."""
+
+import enum
+
+
+class ResultQuality(enum.Enum):
```

Comment:
Is it possible to have an optional numerical factor for results quality instead of/in addition to just bad/good/unknown? For example a goodness-of-fit parameter would be a lot more informative.

Reply:
Good/bad/unknown is what the database supports today. We did talk about allowing any numeric value at one time, but that was dropped because, IIRC, a simple numeric value doesn't give a clear meaning. But we can revisit this if you think it's important.